### PR TITLE
Move jackson modules to toolbox from api-library

### PR DESCRIPTION
**Pull request by MrShish**
Create a new package for our jackson related stuff.. called izettle-jackson

Moved these modules from api-library to this package:

* BigDecimalModule
* InstantRFC3339Module
* UUIDModule

### CODE REVIEW

File: izettle-jackson/pom.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        </dependency>
+        <dependency>
+            <groupId>com.izettle.toolbox</groupId>
+            <artifactId>izettle-java</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.izettle.toolbox</groupId>
+            <artifactId>izettle-java</artifactId>
+            <version>1.0.85-SNAPSHOT</version>
```
Comment (ssprang) [inline izettle-jackson/pom.xml]: Huh? Twice?

File: izettle-jackson/src/main/java/com/izettle/jackson/module/UUIDModule.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.izettle.java.UUIDFactory;
+import java.io.IOException;
+import java.util.UUID;
+
+public class UUIDModule extends SimpleModule {
```
Comment (ssprang) [inline izettle-jackson/src/main/java/com/izettle/jackson/module/UUIDModule.java]: How about a nice descriptive comment here, like for the other modules?

Comment (MrShish) [inline izettle-jackson/src/main/java/com/izettle/jackson/module/UUIDModule.java]: sure!

Comment (MrShish) [inline izettle-jackson/src/main/java/com/izettle/jackson/module/UUIDModule.java]: done

**Comment by ssprang** [comment]
👍

**Comment by xiaodong-izettle** [comment]
Great initiative, @MrShish. 👍 😄

**Pull request merged**